podvm/Makefile.inc

```diff
@@ -80,6 +89,9 @@ SKOPEO_BIN ?= $(SKOPEO_SRC)/bin/skopeo
 
 UMOCI_SRC   = umoci
 
+OPA_SRC = opa
+OPA_BUILD_TARGET := "_release/$(subst v,,$(OPA_VERSION))/opa_linux_$(DEB_ARCH)_static"
```

Comment:
Trying to build this for s390x I'm getting problems where it's trying to use docker and it's not available:
```
1318.2 Cloning https://github.com/open-policy-agent/opa to opa
1318.2 mkdir -p "opa" && cd "opa" && git init && git remote add origin "https://github.com/open-policy-agent/opa" && git fetch --depth=1 origin "v0.58.0" && git reset --hard FETCH_HEAD
1318.2 Initialized empty Git repository in /src/cloud-api-adaptor/podvm/opa/.git/
1328.1 From https://github.com/open-policy-agent/opa
1328.1  * tag               v0.58.0    -> FETCH_HEAD
Updating files: 100% (7736/7736), done.
1329.9 HEAD is now at 69a381c Prepare v0.58.0 release
1329.9 cd "opa" && CC= ARCH=s390x make WASM_ENABLED=0 CGO_ENABLED=0 DOCKER_RUNNING=0 ci-go-ci-build-linux-static
1329.9 make[1]: Entering directory '/src/cloud-api-adaptor/podvm/opa'
1329.9 Docker not installed or not running. Skipping OPA-WASM library build.
1329.9 CGO_ENABLED=0 GOFLAGS="-buildmode=exe" go generate
1338.3 docker run --rm -u 0:0 -v /src/cloud-api-adaptor/podvm/opa:/src -w /src -e GOCACHE=/src/.go/cache -e GOARCH=amd64 -e CGO_ENABLED=0 -e WASM_ENABLED=0 -e FUZZ_TIME=1h -e TELEMETRY_URL= golang:1.21.3-bullseye /bin/bash -c "git config --system --add safe.directory /src && make ci-build-linux-static"
1338.3 /bin/sh: 1: docker: not found
1338.3 make[1]: *** [Makefile:256: ci-go-ci-build-linux-static] Error 127
1338.3 make[1]: Leaving directory '/src/cloud-api-adaptor/podvm/opa'
1338.3 make: *** [Makefile.inc:187: /src/cloud-api-adaptor/podvm/files/usr/local/bin/opa] Error 2
```

Reply:
Hmm, any suggestions on what should be used? I used the settings from the kata PR - https://github.com/kata-containers/kata-containers/pull/7769/files#diff-c48d511a9715d535281eea19171682653656dc361be1547ac1a8cd9bc790edbfR517-R530

Reply:
Looks like docker CLI is must.

Reply:
Yeah, I guess we might just need to add docker to the podvm builder?

Reply:
I found another build target that might work. Can you please recheck

Reply:
Still failing:
```
1298.9 Cloning https://github.com/open-policy-agent/opa to opa
1298.9 mkdir -p "opa" && cd "opa" && git init && git remote add origin "https://github.com/open-policy-agent/opa" && git fetch --depth=1 origin "v0.58.0" && git reset --hard FETCH_HEAD
1298.9 Initialized empty Git repository in /src/cloud-api-adaptor/podvm/opa/.git/
1313.7 From https://github.com/open-policy-agent/opa
1313.7  * tag               v0.58.0    -> FETCH_HEAD
Updating files: 100% (7736/7736), done.
1315.5 HEAD is now at 69a381c Prepare v0.58.0 release
1315.6 cd "opa" && CC= ARCH=s390x make GOOS=linux WASM_ENABLED=0 CGO_ENABLED=0 go-build
1315.6 make[1]: Entering directory '/src/cloud-api-adaptor/podvm/opa'
1315.6 Docker not installed or not running. Skipping OPA-WASM library build.
1315.6 CGO_ENABLED=0 GOFLAGS="-buildmode=exe" go generate
1323.5 CGO_ENABLED=0 GOFLAGS="-buildmode=exe" go build -tags= -o opa_linux_amd64 -ldflags " -X github.com/open-policy-agent/opa/version.Hostname=localhost"
1337.2 make[1]: Leaving directory '/src/cloud-api-adaptor/podvm/opa'
1337.2 install --compare "opa/"opa_linux_s390x"" "/src/cloud-api-adaptor/podvm/files/usr/local/bin/opa"
1337.2 install: cannot stat 'opa/opa_linux_s390x': No such file or directory
1337.2 make: *** [Makefile.inc:188: /src/cloud-api-adaptor/podvm/files/usr/local/bin/opa] Error 1
```
I'll try and have more of a dig into it tomorrow. BTW - I'm not actually running this on s390x, just on x86 in case you wanted to try it out yourself. The command I'm running (the default policy bit might not work) is:
`
ARCH=s390x DEFAULT_AGENT_POLICY_FILE=allow-all-except-exec-process.rego make podvm-builder podvm-binaries podvm-image`

Reply:
Hey Steve, I think the latest fix should work. I built with s390x on x86 and the build succeeded. Please recheck when you find time.

Reply:
I'm getting errors in the build again, but it's building the attestation-agent, not opa related:
```
cd app && RUSTFLAGS=" -C linker=s390x-linux-gnu-gcc" cargo build --release --no-default-features --features " sample_kbc ttrpc,openssl" --target s390x-unknown-linux-gnu
993.5     Updating crates.io index
999.2     Updating git repository `https://gitee.com/anolis/csv-rs`
1130.2 warning: spurious network error (3 tries remaining): failed to connect to gitee.com: Connection timed out; class=Os (2)
1267.4 warning: spurious network error (2 tries remaining): failed to connect to gitee.com: Connection timed out; class=Os (2)
1402.6 warning: spurious network error (1 tries remaining): failed to connect to gitee.com: Connection timed out; class=Os (2)
1539.8 error: failed to get `csv-rs` as a dependency of package `attester v0.1.0 (/guest-components/attestation-agent/attester)`
```

https://gitee.com/ isn't working for me, so I guess this is a network issue on there side. I've tried a couple of times now, but will check the website and re-try later. Sorry for the delay